### PR TITLE
test(ui): Return router props from initializeOrg

### PR DESCRIPTION
**Pull request by scttcper**
Helps with the conversion of view tests to typescript. 

Adds a generic that can correctly type view query parameters.
```ts
// For a component with RouteComponentProps<{alertId: string}, {}>
initializeOrg({router: {params: {alertId: '123'}}})
```

before
```tsx
const {router, route, organization} = initializeOrg()
<Onboarding
  router={router}
  location={router.location}
  params={routeParams}
  routes={router.routes}
  routeParams={router.params}
  route={route}
/>
```

after
```tsx
const {routerProps, organization} = initializeOrg()
<Onboarding {...routerProps} />
```

### CODE REVIEW

File: tests/js/sentry-test/initializeOrg.tsx [hunk 1]
```diff
@@ -46,13 +60,31 @@ export function initializeOrg({
     },
   ]);
 
+  /**
+   * A collection of router props that are passed to components by react-rotuer
```
Comment (priscilawebdev) [inline tests/js/sentry-test/initializeOrg.tsx:64]:
```suggestion
   * A collection of router props that are passed to components by react-router
```

File: tests/js/sentry-test/initializeOrg.tsx [hunk 2]
```diff
@@ -46,13 +60,31 @@ export function initializeOrg({
...
+   *
+   * Pass custom router params like so:
+   * ```ts
+   * initializeOrg({router: {params: {alertId: '123'}}})
+   * ```
+   */
+  const routerProps: RouteComponentProps<RouterParams, {}> = {
+    params: router.params as any,
```
Comment (priscilawebdev) [inline tests/js/sentry-test/initializeOrg.tsx:72]:
```suggestion
    params: router.params as Record<string, any>,
```

🤷‍♀️

**Review by priscilawebdev — APPROVED** [review]
left a few comments but it looks good to me. It was so verbose before and now it is much better! Thanks for doing that 🚀

File: tests/js/sentry-test/initializeOrg.tsx (hunk 2 above)
Comment (scttcper) [reply tests/js/sentry-test/initializeOrg.tsx:72]: it just gets overridden above with `RouteComponentProps<RouterParams, {}>`

**Pull request merged**